packages/graphql-testing/src/MockGraphqlClient.ts

```diff
@@ -0,0 +1,24 @@
+import ApolloClient, {ApolloClientOptions} from 'apollo-client';
+
+import Operations from './Operations';
+
+import MemoryApolloLink from './MemoryApolloLink';
+
+export default class MockGraphQLClient extends ApolloClient<unknown> {
+  readonly graphQLRequests: Operations;
+  private memoryLink: MemoryApolloLink;
+
+  constructor(options: ApolloClientOptions<unknown>) {
+    const memoryLink = new MemoryApolloLink();
+    super({
+      ...options,
+      link: memoryLink.concat(options.link),
```

Comment:
Does this work OK if `options.link` is undefined? Can it be undefined?

Reply:
because I've typed it with `ApolloClientOptions`, link cannot be undefined

Reply:
You can change your options to be something like `Omit<ApolloClientOptions, 'link'>` though, no?

Reply:
so you want the support for undefined. Sure thing.